src/examples/compliance.ts

```diff
@@ -0,0 +1,178 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import {
+  ClaimType,
+  ConditionTarget,
+  ConditionType,
+  ScopeType,
+} from '@polymeshassociation/polymesh-sdk/types';
+import { toHumanReadable } from '@polymeshassociation/polymesh-sdk/utils/internal';
```

Comment:
We shouldn't be exposing this in examples, it's an internal util

Reply:
Added a separate func similar to what we have in rest-api for the same.